### PR TITLE
DZ_1_2

### PR DESCRIPTION
**Pull request by qtyybath**

### CODE REVIEW

**Comment by qtyybath** [comment]
Прошу дати зворотній звязок

File: DZ_1_2.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#домашне завдання 2 заняття
```
Comment (acman) [inline DZ_1_2.py]: nit: add space after `#` in comments
```suggestion
# домашне завдання 2 заняття
```

File: DZ_1_2.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    print(b)
+
+if b > 0:
+    print(a)
+
+if a > b:
+    print(":(")
+else:
+    print(":)")
+
```
Comment (acman) [inline DZ_1_2.py:21]: nice to see blank line at the end!

**Review by acman — APPROVED** [review]
Nice extra effort! It would be great to write branch name in latinic way

**Pull request merged**